.github/pull_request_template.md

```diff
@@ -0,0 +1,22 @@
+# Description
+
+Please include a summary of the changes and the related issue (if any). Please also include relevant motivation and context.
+
+Fixes # (issue)
+
+# Type of change
+
+- [ ] Bug fix (non-breaking change which fixes an issue)
```

Comment:
we should have a commit message check in pre-commit which overlaps with this in theory. Should we fix that to be more reliable? just have both and leave it messy?

Reply:
I don't understand. Both what?

Reply:
Both the pre-commit check asking for semver "conventional commit" message like fix, feat, as well as filling in a form on the PR.

Reply:
what was the resolution? I'm not going to fill in this part of the form if I've already put a proper semantic commit message, and I think others should be doing the same.

Reply:
For silo & repos that will enforce semantic commit messages we can create an override template that removes that section once enforced. I don't believe any of our repos enforce it yet so this is still good to have to help casual contributors describe their PRs better on our OSS repos.